lib/timeline/component/item/BackgroundItem.js

```diff
@@ -49,54 +49,48 @@ BackgroundItem.prototype.isVisible = function(range) {
   return (this.data.start < range.end) && (this.data.end > range.start); 
 };
 
-/**
- * Repaint the item
- */
-BackgroundItem.prototype.redraw = function() {
-  var dom = this.dom;
-  if (!dom) {
+BackgroundItem.prototype._createDomElement = function() {
+  if (!this.dom) {
     // create DOM
     this.dom = {};
-    dom = this.dom;
 
     // background box
-    dom.box = document.createElement('div');
+    this.dom.box = document.createElement('div');
     // className is updated in redraw()
 
     // frame box (to prevent the item contents from overflowing
-    dom.frame = document.createElement('div');
-    dom.frame.className = 'vis-item-overflow';
-    dom.box.appendChild(dom.frame);
+    this.dom.frame = document.createElement('div');
+    this.dom.frame.className = 'vis-item-overflow';
+    this.dom.box.appendChild(this.dom.frame);
 
     // contents box
-    dom.content = document.createElement('div');
-    dom.content.className = 'vis-item-content';
-    dom.frame.appendChild(dom.content);
+    this.dom.content = document.createElement('div');
+    this.dom.content.className = 'vis-item-content';
+    this.dom.frame.appendChild(this.dom.content);
 
     // Note: we do NOT attach this item as attribute to the DOM,
     //       such that background items cannot be selected
-    //dom.box['timeline-item'] = this;
+    //this.dom.box['timeline-item'] = this;
 
     this.dirty = true;
   }
+}
 
-  // append DOM to parent DOM
+BackgroundItem.prototype._appendDomElement = function() {
   if (!this.parent) {
     throw new Error('Cannot redraw item: no parent attached');
   }
-  if (!dom.box.parentNode) {
+  if (!this.dom.box.parentNode) {
     var background = this.parent.dom.background;
     if (!background) {
       throw new Error('Cannot redraw item: parent has no background container element');
     }
-    background.appendChild(dom.box);
+    background.appendChild(this.dom.box);
   }
   this.displayed = true;
+}
 
-  // Update DOM when item is marked dirty. An item is marked dirty when:
-  // - the item is not yet rendered
-  // - the item's data is changed
-  // - the item is selected/deselected
+BackgroundItem.prototype._updateDirtyDomComponents = function() {
```

Comment:
How about retaining the previous comment block as a header comment for this function?

I'm surprised that no header comment passes linting; The linter freaks out if I leave a method uncommented.

Reply:
Ummm I'm not sure how this passes but I assumed if it's not yelling at me, it's ok.